### ISSUE TITLE
Bug - using FAUCET a port configured as tagged is not sending traffic

### ISSUE THREAD

**Issue by Dru90** [NorthboundNetworks/ZodiacFX#92]
When using the Zodiac FX with the FAUCET controller:

The flow installed on the ZodiacFX to handle traffic coming in on port 3 (flow 18), VLAN 200 tells me no bytes are matching this flow despite my networking sending traffic on this vlan ID to this port. Interestingly enough I do see traffic go from the untagged ports (flow 17) to the tagged port, just no traffic in the other direction. Flows and Faucet yaml config file below:

```
    zodiacfx:
        dp_id: 0x************
        description: "ZodiacFX"
        hardware: "ZodiacFX"
        interfaces:
            1:
                native_vlan: 200
            2:
                native_vlan: 200
            3:
                tagged_vlans: [200]

 Flow 17
  Match:
  In Port: 2

 Attributes:
  Table ID: 0				Cookie:0x5adc15c0
  Priority: 9000				Duration: 162 secs
  Hard Timeout: 0 secs			Idle Timeout: 0 secs
  Byte Count: 9720			Packet Count: 162
  Last Match: 00:00:00

 Instructions:
  Apply Actions:
   Push VLAN tag
   Set VLAN ID: 200
  Goto Table: 1

_______

 Flow 18
  Match:
  In Port: 3
  VLAN ID: 200

 Attributes:
  Table ID: 0				Cookie:0x5adc15c0
  Priority: 9000				Duration: 162 secs
  Hard Timeout: 0 secs			Idle Timeout: 0 secs
  Byte Count: 0			Packet Count: 0
  Last Match: 00:02:42

 Instructions:
  Goto Table: 1
```

I came across this bug when attempting to trunk a Zodiac FX to a Zodiac GX.

Dru

**Comment by tweippert** [comment]
I see the same behaviour when trying to trunk between OpenVswitch and Zodiac FX.

Is there some new information ?

thanks, 
tim

**Comment by pzanna** [comment]
This should now be fixed in v0.86, please let me know if it continues.
